setup.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from setuptools import Command, setup
+from setuptools import find_packages
+from unittest import TestLoader, TextTestRunner
+
+from os import environ, path
+
+PROJECT = 'oca-decorators'
+SHORT_DESC = (
+    'This is a library of decorators for Odoo developers.'
+)
+README_FILE = 'README.rst'
+
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: GNU Lesser General Public License v3 or '
+    'later (LGPLv3+)',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+]
+
+version = environ.get('RELEASE') or environ.get('VERSION') or '0.0.1'
+
+if environ.get('TRAVIS_BUILD_NUMBER'):
+    version += 'b%s' % environ.get('TRAVIS_BUILD_NUMBER')
```

Comment:
Please don't hardcode travis. I already hit it with MQT. It's better IMHO if you provide a vendorless variable and transform the travis one into that in `.travis.yml`.

Reply:
I need something to guarantee an increasing build number. Open to ideas, but this is the one I have.

Reply:
It's worth noting this works perfectly fine without Travis. It's required to push to PyPi with guaranteed versions. I've never actually been able to get `tox` to work, so this is what we have.

I'm pretty sure this logic can be removed if someone submits a PR implementing `tox`. I've spent way too much time trying to figure it out in the past, with zero luck, so I'm not willing to look again. Sorry.

Reply:
@lasley The best way is to use `setuptools_scm`to manage the version. It's what we use for git-aggregator for exemple. https://github.com/acsone/git-aggregator/commit/545a09728895aa827e78a5cd7519b22e5f94ca4c#diff-2eeaed663bd0d25b7e608891384b7298
As you can see in the previous link, you can also use travis to deploy the lib each time you create a tag.

Reply:
Thanks for the example @lmignon - much easier than Tox. Totally looks doable.

That `deploy` section in the Travis file - is that provided by `setuptools_scm`?

Reply:
Oh wow nvm this is just a TravisCI feature. Well that's freaking cool